Updating script, images list, documentation for new versions of opens…

etsauer opened the pull request:
…hift

#### What does this PR do?
Updating registry sync script and image lists for provide image lists for `3.3.1.11` and `3.4.1.10`

#### How should this be manually tested?
follow solutions/disconnected_installs/README.md

#### Is there a relevant Issue open for this?
n/a

#### Who would you like to review this?
/cc @oybed @sabre1041 @stratus-ss

etsauer committed:
Updating script, images list, documentation for new versions of openshift

sabre1041 commented:
@etsauer Sync works well.

A couple notes:

1 Any way to add a counter as it is progressing through the list of images to sync. Such as "Syncing Image 3/25" to provide some additional insight into the progress. 
2. Link in `README.md` is in asciidoc format instead of markdown
3. Should the "Create new OpenShift images file" be worded differently? It seems to imply that you must create a new file even though we provide a number of versions included in the repository

etsauer committed:
Adding a counter and refactoring README to make more sense

etsauer commented:
@sabre1041 check out the changes i pushed. I added a counterof images completed. Note that the implementation of the counter is different depending on if you use `--parallel` or not. Also addressed your README concerns.
